xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsApplicationsPreferencesMacro.xml

```diff
@@ -1044,7 +1045,12 @@ require(['jquery', 'xwiki-meta', 'ApplicationWidget', 'xwiki-bootstrap-switch',
       #end
       ## Collapse button
       &lt;td class="notificationCollapseButtonCell"&gt;
-        &lt;button type="button" class="btn btn-default btn-sm collapseButton"&gt;$services.icon.renderHTML('caret-down')&lt;/button&gt;
+        &lt;button id="notifications-applications-preferences-macro-$appCount-button" type="button" class="btn btn-default btn-sm collapseButton"&gt;
+          $services.icon.renderHTML('caret-down')
+        &lt;/button&gt;
+        &lt;label for="notifications-applications-preferences-macro-$appCount-button" class='sr-only'&gt;
+        $escapetool.xml($services.localization.render('notifications.settings.applications.table.dropdown.toggle.label',[$type.applicationName]))
+        &lt;/label&gt;
```

Comment:
Why don't you just put the content inside the button in a `<span>`? I don't think it's a best practice to associate a label with a button.

Reply:
I think keeping the translation as "[...].dropdown.toggle.label" is correct. This could need to change with this change though.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2217/commits/344f06dcf537054a0e718adbd66fdaf09482d199